### PR TITLE
Add highlights for the week ending August 6, 2021

### PR DESCRIPTION
**Pull request by marians**

### CODE REVIEW

**Comment by marians** [comment]
Postponed by one week

**Comment by marians** [comment]
@cokiengchiara Please send in app related content until 14:00. Thank you!

File: src/content/changes/highlights/2021-08-06.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+The unique, immutable cluster identifier formerly called the cluster ID is now called the cluster name, and what was called name is now called description. The goal here is consistency between our user interfaces and the Management API. Likewise for node pools.
+
+## Apps
+
+[Linkerd2 v0.6.0](https://docs.giantswarm.io/changes/managed-apps/linkerd2-app/v0.6.0/), our managed service mesh offering, is now available as an app in the Giant Swarm Catalog. To try it, please read the [documentation](https://github.com/giantswarm/linkerd2-app).
```
Comment (cokiengchiara) [inline src/content/changes/highlights/2021-08-06.md:16]: This PR https://github.com/giantswarm/docs/pull/1073 needs to be merged before link will work

File: src/content/changes/highlights/2021-08-06.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+The unique, immutable cluster identifier formerly called the cluster ID is now called the cluster name, and what was called name is now called description. The goal here is consistency between our user interfaces and the Management API. Likewise for node pools.
+
+## Apps
```
Comment (cokiengchiara) [inline src/content/changes/highlights/2021-08-06.md:14]: @ubergesundheit , @piontec , can you check the apps section to make sure you are happy with it? 🙏

**Review by piontec — APPROVED** [review]
LGTM for apps

File: src/content/changes/highlights/2021-08-06.md (hunk 1 above)
Comment (marians) [reply src/content/changes/highlights/2021-08-06.md:16]: Approved. Let's then also wait until https://github.com/giantswarm/docs/pull/1071 picks it up and then approve that one.

File: src/content/changes/highlights/2021-08-06.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+[Linkerd2 v0.6.0](https://docs.giantswarm.io/changes/managed-apps/linkerd2-app/v0.6.0/), our managed service mesh offering, is now available as an app in the Giant Swarm Catalog. To try it, please read the [documentation](https://github.com/giantswarm/linkerd2-app).
+
+[Prometheus Operator v0.10.0](https://docs.giantswarm.io/changes/managed-apps/prometheus-operator-app/v0.10.0/) is upgraded to upstream Kube Prometheus Stack v17.0.3.
```
Comment (ubergesundheit) [inline src/content/changes/highlights/2021-08-06.md:18]:
```suggestion
[Prometheus Operator v0.10.0](https://docs.giantswarm.io/changes/managed-apps/prometheus-operator-app/v0.10.0/) is upgraded to upstream Kube Prometheus Stack v17.0.3 and now contains prometheus-operator [v0.49.0](https://github.com/prometheus-operator/prometheus-operator/releases/tag/v0.49.0), prometheus [v2.28.1](https://github.com/prometheus/prometheus/releases/tag/v2.28.1) and alertmanager [v0.22.2](https://github.com/prometheus/alertmanager/releases/tag/v0.22.2).
```

**Review by cokiengchiara — APPROVED** [review]
Overall content LGTM! I didn't check for comprehensiveness of non-apps section, but I trust Marian did that.

File: src/content/changes/highlights/2021-08-06.md (hunk 1 above)
Comment (marians) [reply src/content/changes/highlights/2021-08-06.md:16]: That's done.

Comment (cokiengchiara) [reply src/content/changes/highlights/2021-08-06.md:16]: Hmmm. Linkerd already disappeared from Playground https://docs.giantswarm.io/changes/playground-apps/, but it's still not in Managed Apps https://docs.giantswarm.io/changes/managed-apps/

Comment (cokiengchiara) [reply src/content/changes/highlights/2021-08-06.md:16]: Ahhh. Just merged this one https://github.com/giantswarm/docs/pull/1075#pullrequestreview-724267825

**Pull request merged**